module-3/web-app/src/components/Pages/Projects/Projects.jsx

```diff
@@ -1,30 +1,35 @@
 import React, {useState, useEffect} from "react"
+import {Link} from "react-router-dom"
 import style from "./Projects.module.scss"
 
 //TODO in progress projects catalogue page
 const Projects = () => {
     const [projects, setProjects] = useState('')
+    const [documents, setDocuments] = useState('')
    
     useEffect(() => {
-        async function getData(url = "http://localhost:3001/selectAllProjects") {
-            const response = await fetch(url, {
-                method: 'GET'
-            })
-
-            return response.json() 
-        }
-        getData().then(res => setProjects(res))
+        fetch("http://localhost:3001/selectAllProjects/")
```

Comment:
Add TODO comment to move this in component fetch to a redux action later on

Reply:
Done